README.md

```diff
@@ -22,29 +22,31 @@ Docstrings are automatically inserted from the GDAL documentation. Note that the
 ```
 julia> using GDAL
 
-help?> GDAL.creategeometry
+help?> GDAL.ogr_g_creategeometry
   OGR_G_CreateGeometry(OGRwkbGeometryType eGeometryType) -> OGRGeometryH
 
   Create an empty geometry of desired type.
 
-     Parameters
-    ––––––––––––
+  Parameters
+  ––––––––––––
 
-    •  eGeometryType: the type code of the geometry to be created.
+    •    eGeometryType: the type code of the geometry to be created.
 
-     Returns
-    –––––––––
+  Returns
+  –––––––––
 
-  handle to the newly create geometry or NULL on failure. Should be freed with
-  OGR_G_DestroyGeometry() after use.
+  handle to the newly create geometry or NULL on failure. Should be freed with OGRGDestroyGeometry() after use.
 ```
 
 Further usage documentation is not yet available, but the files `test/tutorial_raster.jl` and `test/tutorial_vector.jl` should provide a good hint based on the API tutorials from [GDAL.org](http://gdal.org/).
 
 The bulk of this package is generated automatically by the scripts under `gen/`. For developer documentation regarding this process see `gen/README.md`.
 
-## Low-level and mid-level API
-Next to the functions available in the `GDAL` module, there is also a submodule `GDAL.C` available. `GDAL.C` is the low-level wrapping, without any error checking, function renaming or type wrapping. If you know what you are doing and don't want this package to get in your way, use this. Currently the `GDAL.C` module provides a more complete wrapping, though the intention is to make the complete user facing C API available in both modules.
+## Changelog
+
+Since GDAL.jl v0.2.0 the package has changed considerably. The GDAL C function names are no longer being shortened. This brought many complications forcing us to simulate GDAL's types to disambiguate methods. As an example, the old `GDAL.open` now has to be made explicit: use `GDAL.gdalopen`, `ogr_dr_open`, or `ogropen`. Other options are simpler, `GDAL.allregister` can only become `GDAL.gdalallregister`. To help with updating old code in a semi automatic fashion, look at [this script] which uses [MacroTools.jl](https://mikeinnes.github.io/MacroTools.jl/stable/sourcewalk/) to do function renaming.
```

Comment:
should be: `[this script](https://gist.github.com/visr/0a2ad3fe92073345c93c2ca42f5f58a0#file-renamer-jl)`

Reply:
Good catch, thanks!